### DISCUSSION TITLE
Review code @mezgoodle

### DISCUSSION THREAD

**Discussion by mezgoodle** [HowProgrammingWorks/LiveQA#2] [Code review]
Я вже зробив проект, і надіслав його у форму, але поки є час, то нехай буде рев'ю. Можливо, щось покращу.

Посилання на [проект](https://github.com/mezgoodle/meziblog).

**Comment by tshemsedinov** [comment]
Everything is ok, just one issue, see this: https://github.com/mezgoodle/meziblog/issues/15

**Comment by tshemsedinov** [comment]
We have video recording of 1st review, see in telegram of your group
